homeassistant/components/sensibo/coordinator.py

```diff
@@ -39,7 +40,7 @@ async def _async_update_data(self) -> dict[str, dict[str, Any]]:
         try:
             for dev in await self.client.async_get_devices():
                 devices.append(dev)
-        except (pysensibo.SensiboError) as error:
+        except (AuthenticationError, SensiboError) as error:
```

Comment:
We should raise `ConfigEntryAuthFailed` instead and implement a reauth flow

Reply:
As I wrote in the PR description that would be next step. Should we put that in this PR?